yamanifest/yamf.py

```diff
@@ -37,6 +37,7 @@ def parse_args(args):
     parser_add = subparsers.add_parser('add', help='Add filepaths to manifest')
     parser_add.add_argument('-n','--name', default='manifest.yaml', action='store', help='Manifest file name')
     parser_add.add_argument("-f","--force", help="Force overwrite of existing manifest", action='store_true')
+    parser_add.add_argument("-u", "--update", help="Update entries of existing manifest", action="store_true")
```

Comment:
This option was added but appears to be unused?

Reply:
Whoops! part of a different solution I was thinking about :D

Reply:
Removed in https://github.com/ACCESS-NRI/yamanifest/pull/18/commits/02b2af6176716c0df82765aa13fb530fc46d5b7d